packages/braid-design-system/project.json

```diff
@@ -1,25 +1,54 @@
 {
   "$schema": "https://raw.githubusercontent.com/nrwl/nx/15.4.2/packages/nx/schemas/project-schema.json",
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*"]
+  },
   "targets": {
     "build": {
-      "dependsOn": ["copy-codemod", "generate-icons"],
+      "dependsOn": ["codemod", "generate:icons"],
       "executor": "nx:run-script",
       "options": {
         "script": "build"
       }
     },
-    "copy-codemod": {
+    "codemod": {
       "dependsOn": ["^build"],
+      "inputs": ["^default"],
+      "outputs": ["{projectRoot}/codemod/dist"],
+      "executor": "nx:run-script",
+      "options": {
+        "script": "codemod"
+      }
+    },
+    "generate:icons": {
+      "inputs": ["{projectRoot}/icons/*.svg", "{projectRoot}/scripts"],
+      "outputs": ["{projectRoot}/src/**/*Svg.tsx"],
```

Comment:
I think it's worth scoping this to specifically the `icons` folder, just in case there's ever some other file that ends with `Svg.tsx`.